lib/src/main/java/com/thenewboston/utils/Pagination.kt

```diff
@@ -0,0 +1,33 @@
+package com.thenewboston.utils
+
+data class PaginationOptions(
+    val offset: Int,
+    val limit: Int
+) {
+    fun toQuery() = "?offset=$limit&limit=$limit"
```

Comment:
Should offset not be set to $offset ?

Reply:
Yes it should. 
My bad. 
I will change that now